buildbot/slave/files/config/buildbot.tac

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
I believe the `.tac` files are generated by Buildbot, so I would prefer to leave them untouched.